### ISSUE TITLE
[Bug]: Blockquotes aren't styled appropriately

### ISSUE THREAD

**Issue by honor-gfsc** [geeksforsocialchange/gfsc-v3#230]
## Description
Blockquotes are too small, font wise, which looks broken

Image to illustrate:
<img width="725" alt="Screenshot 2023-04-17 at 15 47 41" src="https://user-images.githubusercontent.com/127757676/232525691-c5a5cec6-1b40-4166-8592-d208a88cfd2c.png">

currently failing WCAG AA for contract `3.1:1`

## acceptance criteria

- [ ] pull quotes meet WCAG AA

## plan / notes

- increase fontsize and till it is "large text" as defined by [WCAG](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#larger-scaledef) 1.2-1.5 * body text size
- If this doesn't look good change color back to same as body but keep orange side bar
- longest quote in https://gfsc.studio/blog/2016/no-false-users/

**Comment by kimadactyl** [comment]
Just a note its always really useful if you can link where you find things so we don't have to go hunting for it :)

**Comment by aaaaargZombies** [comment]
The change to design if any should be agreed and implemented before work on #194 is started
